### DISCUSSION TITLE
Eza is not showing symlinks!

### DISCUSSION THREAD

**Discussion by eithanunal** [eza-community/eza#1170] [Q&A, Howto, Help]
When i use "ls -la" if there's a symlink ls displays (/.../... -> /.../...) but i've been using eza -la and it never shows (->) maybe i should use other flags but i don't find any others!

**Comment by cafkafk** [comment]
it seems to work fine for me, what version are you on?

**Reply by eithanunal** [reply]
I re-installed eza and it worked  fine. thank you
